tests/dateSelector.spec.ts

```diff
@@ -10,97 +10,35 @@ test.beforeEach(async ({ page }) => {
 
 test('Select the desired date in the calendar', async ({ page }) => {
     const pm = new PageManager(page)
-    await pm.onOwnerInformationPage().openFormAddNewPetForOwner('Harold Davis')
+    await pm.onOwnersPage().openOwnerInfoPageFor('Harold Davis')
+    await pm.onOwnerInformationPage().openFormAddNewPet()
 
-    const date = {
-        searchYear: '2014',
-        searchMonth: '05',
-        searchDay: '02'
+    const { birthYear, birthMonth, birthDay } = {
+        birthYear: '2014',
+        birthMonth: '05',
+        birthDay: '02'
     }
-    
-    await pm.onOwnerInformationPage().addPet('Tom', date, 'dog')
 
-    const petItem = page.locator('td', { has: page.getByText('Tom') })
-    await expect(petItem).toBeVisible();
-    await expect(petItem).toContainText(`${date.searchYear}-${date.searchMonth}-${date.searchDay}`)
-    await expect(petItem).toContainText('dog')
-
-    await petItem.getByRole('button', { name: "Delete Pet" }).click()
-    await expect(petItem).toHaveCount(0)
+    await pm.onAddNewPetPage().addPet('Tom', birthDay, birthMonth, birthYear, 'dog')
+    const pet = await pm.onOwnerInformationPage().validatePetInfo('Tom', birthDay, birthMonth, birthYear, 'dog')
```

Comment:
The constant name can be better.
By saying "pet" I thought it's a pet object, but it's actually only value of "petName"